defender/connection.py

```diff
@@ -13,14 +12,18 @@
 urlparse.uses_netloc.append("redis")
 
 
-MOCKED_REDIS = mockredis.mock_strict_redis_client()
+MOCKED_REDIS = None
 INVALID_CACHE_ERROR_MSG = 'The cache {} was not found on the django cache' \
                           ' settings.'
 
 
 def get_redis_connection():
     """ Get the redis connection if not using mock """
     if config.MOCK_REDIS:  # pragma: no cover
+        global MOCKED_REDIS
+        if MOCKED_REDIS is None:
+            import mockredis
+            MOCKED_REDIS = mockredis.mock_strict_redis_client()
```

Comment:
We can probably make this even more simple.

```
if config.MOCK_REDIS:
    import mockredis
    return mockredis.mock_strict_redis_client()
```

Then we can remove the `MOCKED_REDIS` variable all together what do you think?

Reply:
Makes sense.

Please feel free to pick up this PR and/or do it from scratch - I've decided to not use django-defender, so will not update it myself.